sos/collector/transports/local.py

```diff
@@ -41,6 +41,9 @@ def _retrieve_file(self, fname, dest):
     def _format_cmd_for_exec(self, cmd):
         return cmd
 
+    def _copy_file_to_remote(self, fname, dest):
+        pass
```

Comment:
Hmm. I was initially going to say this should return `True`, but that might not track given that we may not be writing the the same location (and in this first implementation we already aren't).

This might need some attention later one depending on how much use this functionality sees, but for now I think it's fine.

Reply:
Unsure if by same location you mean to the `/tmp/sos.XXXX` path where report does all its temporary work? 

I most often follow the Google Style guide, with typing and return syntax (i.e. `def _copy_file_to_remote(self, fname: str, dest: str) -> bool:`. Since that isn't part of sos style I just leave it out which made it easy to forget about the consistent function return.

FWIW for control persist testing I saw no problem when it was pass, but now that you point it out I'd probably opt for the consistent return among all transports. Unless you see an issue with making it `return True`?